editor/editor_node.cpp

```diff
@@ -624,6 +625,10 @@ void EditorNode::_notification(int p_what) {
 			editor_data.clear_edited_scenes();
 		} break;
 
+		case Control::NOTIFICATION_THEME_CHANGED: {
+			scene_tab_add_ph->set_custom_minimum_size(scene_tab_add->get_minimum_size());
+		} break;
```

Comment:
@YeldhamDev does this actually get called? It doesn't look like it should as `EditorNode` isn't a control and `Control::NOTIFICATION_THEME_CHANGED` is normally only sent to Control nodes.

Reply:
Probably not, likely an oversight of mine.

Reply:
For the record, I'm changing this in https://github.com/godotengine/godot/pull/75472.